apps/condo/pages/_app.tsx

```diff
@@ -114,7 +100,46 @@ const MenuItems: React.FC = () => {
         path: 'miniapps',
         icon: BarMiniAppsIcon,
         label: 'menu.MiniApps',
-    }]
+    },
+]
+const accessConfig = [
```

Comment:
Am I understanding right that this array's elements order must match to items of the `listConfig` array?
Why didn't you add an `access` field to `listConfig`?

Reply:
relying on array indexes is not safe at all